implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -322,9 +322,15 @@ public static <A, T> Converter<A> newArrayConverter(Converter<? extends T> itemC
      * @param <V> the type of the values
      * @return the new converter (not {@code null})
      */
+    @Deprecated
```

Comment:
Why deprecate this one?

Reply:
For the collection converter, we don't offer an API method without the factory, so I wanted to make the API consistent. I guess we could also add a method API that defaults the collection type like the map converter.